domain_access/domain_access.module

```diff
@@ -5,12 +5,13 @@
  * Domain-based access control for content.
  */
 
-use Drupal\domain\DomainNegotiator;
+use Drupal\domain\DomainManager;
```

Comment:
This is outdated. We switched to using a Negotiator class a while back to be consistent with core naming conventions.